src/pluggy/_tracing.py

```diff
@@ -29,24 +28,26 @@ def format_message(self, tags, args):
             lines.append("%s    %s: %s\n" % (indent, name, value))
         return lines
 
-    def processmessage(self, tags, args):
-        if self.writer is not None and args:
-            lines = self.format_message(tags, args)
-            self.writer("".join(lines))
+    def _processmessage(self, tags, args):
```

Comment:
breaking change, but I checked and neither `pytest`, `tox`, or `devpi` use it (and they shouldn't anyway), so I think this is fine.

Reply:
Yes, I checked this too.